pkg/util/json/tokenizer/scanner.go

```diff
@@ -251,7 +252,7 @@ func (s *Scanner) parseStringSlow(pos int) ([]byte, int) {
 			case -1:
 				return nil, pos // Error
 			case 0:
-				break
+				break loop
```

Comment:
yikes.  Good catch.

Reply:
How about just returning nil, pos? no need for loop label then...
I think this "optimization" about eof was not great.

Reply:
Done